### ISSUE TITLE
how to fine-tune from the original pretrained model?

### ISSUE THREAD

**Issue by Kewenjing1020** [zhengyang-wang/Deeplab-v2--ResNet-101--Tensorflow#28]
I try to use original ImageNet pre-trained ResNet models to finetune, but I get the problem that "NotFoundError (see above for traceback): Key resnet_v1_50/block1/unit_1/bottleneck_v1/conv2/BatchNorm/beta not found in checkpoint".
Could anybody tell me how to solve this problem?
